index.html

```diff
@@ -19,8 +19,8 @@
     <header class="header-section flex">
       <nav class="nav-bar">
         <button type="button" class="menu-btn">
-          <img class="hidden" src="./assets/icons/menu.svg" alt="menu" />
-          <img src="./assets/menu/cancel.svg" alt="cancel" style="color: red;"/>
+          <img src="./assets/icons/menu.svg" alt="menu" />
+          <img class="hidden" src="./assets/menu/cancel.svg" alt="cancel" style="color: red;"/>
```

Comment:
I see you added the menu button back into the webpage as requested by the previous reviewer; well done 👍. However, it seems you introduced another bug because the cancel button is no longer showing😅. 

**Hint:** You may need to double-check your javascript file to make sure that the cancel button shows when the menu button is pressed and vice versa